perceval/converters/resources_estimator.py

```diff
@@ -0,0 +1,90 @@
+# MIT License
+#
+# Copyright (c) 2022 Quandela
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# As a special exception, the copyright holders of exqalibur library give you
+# permission to combine exqalibur with code included in the standard release of
+# Perceval under the MIT license (or modified versions of such code). You may
+# copy and distribute such a combined system following the terms of the MIT
+# license for both exqalibur and Perceval. This exception for the usage of
+# exqalibur is limited to the python bindings used by Perceval.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from perceval.converters.circuit_to_graph_converter import CircuitToGraphConverter, gates_and_qubits
+import numpy as np
+from qiskit import QuantumCircuit
```

Comment:
This imposes `qiskit` as a primary 3rd party for Perceval. However, it's just an "extra" requirement. So, the import shouldn't be done at the top level of this class.

What we need: `import perceval` should keep working, without `qiskit` being installed on the venv.

Reply:
Indeed I don't need qiskit here, I just have it to define the circuit as a qiskit_circuit type. Then, can I just remove it and write something like 

 def __init__(self, qiskit_circuit, encoding: list[list[int]] = None):
        """
       :param qiskit_circuit: Qiskit quantum circuit to estimate resources for.
       :param encoding: Custom encoding to use.
        """ 
 ?